docs/sql-ref-datetime-pattern.md

```diff
@@ -177,6 +191,12 @@ Spark uses the below letters in date and timestamp parsing and formatting:
   <td> offset-Z </td>
   <td> +0000; -0800; -08:00; </td>
 </tr>
+<tr>
+  <td> <b>p</b> </td>
+  <td> pad next field with spaces </td>
```

Comment:
Should we also add `Pad modifier` in the detail section if we decided to announce the support for `p`?

Reply:
+1, or we can forbid it if it's hard to document. I never used `p` before...